src/main/java/team/gif/robot/Robot.java

```diff
@@ -18,8 +23,19 @@
     public static final boolean isCompBot = false;
 
     private Command m_autonomousCommand;
-
     private RobotContainer m_robotContainer;
+    public static Pigeon m_pigeon = null;
+    public static NetworkTableEntry examplentry;
+
+    // T.S: Creating an new tab in shuffleboard.
+    ShuffleboardTab tab = Shuffleboard.getTab("FRC2022 test");
+    //public double inputValue;
+    public ShuffleboardInput inputCommand;
```

Comment:
clarify variable name
```suggestion
    public ShuffleboardInput exampleShuffleleboardEntryCommand;
```